pkg/logs/input/tailer/tailer.go

```diff
@@ -148,6 +152,7 @@ func (t *Tailer) forwardMessages() {
 		origin := message.NewOrigin(t.source)
 		origin.Identifier = identifier
 		origin.Offset = offset
+		origin.SetTags([]string{fmt.Sprintf("filename:%s", filepath.Base(t.path))})
```

Comment:
Wondering whether we should remove the extension too ?

Reply:
good point, @NBParis ?

Reply:
great thanks!